packages/oauth/src/callback-options.ts

```diff
@@ -4,6 +4,39 @@ import { InstallURLOptions } from './install-url-options';
 import { Installation, OrgInstallation } from './installation';
 
 export interface CallbackOptions {
+
+  /**
+   * An additional logic to run right before executing the Slack app installation with the given code parameter.
+   *
+   * When this method returns false, the InstallProvider skips the installation.
+   * You can set false when the visiting user is not eligible for proceeding the Slack app installation flow.
+   *
+   * Also, when returning false, this method is responsible to call response#end() method
```

Comment:
```suggestion
   * Also, when returning false, this method is responsible for calling the response#end() method
```

Reply:
Quick question on this one: should it be "is responsible for calling the response#end() method" or "is responsible for calling the callbackRes#end() method"? Like, should the comment refer to the argument name provided the method?